### PR TITLE
OSD improvements

### PR DESCRIPTION
**Pull request by hydra**

### CODE REVIEW

File: src/main/io/osd.c [hunk 1]
```diff
@@ -521,22 +533,15 @@ void osdInit(displayPort_t *osdDisplayPortToUse)
...
-                displayWriteChar(osdDisplayPort, j, i, x++);
-        }
-    }
+    osdDrawLogo(3, 1);
 
     char string_buffer[30];
-    sprintf(string_buffer, "BF VERSION: %s", FC_VERSION_STRING);
-    displayWrite(osdDisplayPort, 5, 6, string_buffer);
+    sprintf(string_buffer, "V%s", FC_VERSION_STRING);
+    displayWrite(osdDisplayPort, 20, 6, FC_VERSION_STRING);
```
Comment (mikeller) [inline src/main/io/osd.c]: Should be `string_buffer`.

File: src/main/io/osd.c [hunk 2]
```diff
@@ -521,22 +533,15 @@ void osdInit(displayPort_t *osdDisplayPortToUse)
...
     displayClearScreen(osdDisplayPort);
 
-    // display logo and help
-    char x = 160;
-    for (int i = 1; i < 5; i++) {
-        for (int j = 3; j < 27; j++) {
-            if (x != 255)
-                displayWriteChar(osdDisplayPort, j, i, x++);
-        }
-    }
```
Comment (mikeller) [inline src/main/io/osd.c]: Should be `osdDrawLogo(osdDisplayPort, 3, 1)`.

**Comment by hydra** [comment]
@mikeller this is a work-in-progress branch which I will force push to periodically while I do some cleanup and merges from the cleanflight osd code.  I was interrupted by the G/F and just pushed un-finished code earlier, hence the do-not-merge label :smile:

File: src/main/io/osd.c (hunk 2 above)
Comment (hydra) [inline src/main/io/osd.c]: Yeah, I was wondering wether or not to update the `display` api to provide a 'displayDrawBlock' or similar method, at this time I decided not to and to keep the logo code in the `osd` api while I discover more about the new code.

**Comment by mikeller** [comment]
Ha, maybe we need a 'work in progress' label. I understood the 'don't merge' label to be 'needs discussion / testing first'.

Or maybe a 'GF' label. :-P

**Comment by hydra** [comment]
@mikeller I will add the work-in-progress label

**Pull request merged**